Dockerfile

```diff
@@ -1,20 +1,12 @@
-FROM golang:1.13 AS build
-
-WORKDIR /sloop
-COPY go.mod go.sum ./
-
-RUN go mod download
-
-COPY pkg ./pkg
-
-RUN curl -o /sloop/aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.14.6/2019-08-22/bin/linux/amd64/aws-iam-authenticator \
+FROM alpine as build
+RUN apk add --no-cache curl
+RUN curl -o /aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.14.6/2019-08-22/bin/linux/amd64/aws-iam-authenticator \
```

Comment:
why do we have this path change?

Reply:
@sana-jawad Initially in the docker file we were using two images one to build the binary and second is the actual sloop image. As the build process is entirely taken care by goreleaser we just moved to the root path directly. instead of getting build data from other image